charts/stable/zwavejs2mqtt/values.yaml

```
@@ -28,6 +28,9 @@ service:
     ports:
       http:
         port: 8091
+      websocket:
+        enabled: true
```

```suggestion
        enabled: false
```

Not everyone would want this enabled.